typegate/tests/e2e/typegraph/typegraphs/python/multiple_runtimes.py

```diff
@@ -0,0 +1,29 @@
+from typegraph_next import t, typegraph, Graph
+from typegraph_next.policy import Policy
+from typegraph_next.runtimes.python import PythonRuntime
+from typegraph_next.runtimes.deno import DenoRuntime
+
+
+@typegraph()
+def test_multiple_runtimes(g: Graph):
+    public = Policy.public()
+    deno = DenoRuntime()
+    python = PythonRuntime()
+
+    g.expose(
```

Comment:
no test using this typegraph ?

Reply:
It is used in test `typegraph_test.ts`, `testSerializeAll`.